maintainers/maintainer-list.nix

```diff
@@ -6529,12 +6529,6 @@
     githubId = 3359345;
     name = "obadz";
   };
-  ocharles = {
```

Comment:
Do you plan to add yourself in the foreseeable future again? If so please do not remove yourself.

Reply:
No